scripts/client/lint-rust.mjs

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env zx
 import 'zx/globals';
-import { workingDirectory } from '../utils.mjs';
+import { getClippyToolchain, workingDirectory } from '../utils.mjs';
 
-// Check the client using Clippy.
+// Check the client using nightly clippy.
```

Comment:
Maybe the comment should something like say "using the specified toolchain (or `stable` by default) clippy".